mthamayor/auto-mart
# #166738957 Added database relation to flags endpoint

#### What does this PR do?
Implement database requirement to 'flags' endpoints

#### Description of Task to be completed?
Switch from using dummyDB to a persistent Postgres database for the following endpoints
1. **POST /flag/**

#### How should this be manually tested?
- Clone the repo and CD into it
- Install dependencies with `npm install`
- On your machine, create a database named **'auto_mart'** and make sure the user **'postgres'** has full privileges
- You can now test the above-specified endpoints on postman
Check [the documentation](https://mthamayor-auto-mart.herokuapp.com/api/v1/docs) for the endpoints specifications

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166738957

#### Screenshots (if appropriate)
None

#### Questions:
None
